lib/route.js

```diff
@@ -955,6 +955,23 @@ function route(harvester, name, model, schema, routeOptions) {
     }
 
     this.appendLinks = appendLinks;
+
+    function disallow(methods) {
+        _.forEach(methods, function (methodName) {
+            var method = _this[methodName]();
+            method.options.notAllowed = true;
+            method.register();
```

Comment:
We wanted the .register call so that you could externally decide whether or not a route gets registered- otherwise you have little control over the order that routes get created and registered in. I think we did a bad job of communicating this, so register wound up getting called inside of harvester, which makes it sort of worthless for the moment. I think the preferred behaviour is that there is an additional property, or that you rely on some configuration that lets you know which routes should be registered and which should not be